### PR TITLE
Add AWS examples

### PR DESCRIPTION
**Pull request by m110**

### CODE REVIEW

File: _examples/pubsubs/aws-sns/main.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
+// Sources for https://watermill.io/docs/getting-started/
```
Comment (roblaszczak) [inline _examples/pubsubs/aws-sns/main.go:1]: I guess you can add i to to getting-started as well?

Comment (m110) [reply _examples/pubsubs/aws-sns/main.go:1]: bd3b9b253a88f4d57b09c6d69ef22fddd0eeb7cb

**Review by roblaszczak — APPROVED** [review]

File: docs/content/docs/getting-started.md [hunk 2]
```diff
@@ -198,6 +198,24 @@ A more detailed explanation of how it is working (and how to add live code reloa
 {{% load-snippet-partial file="src-link/_examples/pubsubs/sql/main.go" first_line_contains="func process" %}}
 {{< /tab >}}
 
+{{< tab "AWS SQS" "aws-sqs" >}}
```
Comment (roblaszczak) [inline docs/content/docs/getting-started.md:201]: sns missing?

Comment (m110) [reply docs/content/docs/getting-started.md:201]: c87302be8c9d8f039f8e80d9126dde4c90e06169

**Pull request merged**